### PR TITLE
Update r2yara URL

### PR DESCRIPTION
**Pull request by sylvainpelissier**
Update r2yara repository with the new URL.

**Checklist**

- [ ] Closing issues: #issue
- [x] Mark this if you consider it ready to merge
- [ ] I've added tests (optional)
- [ ] I wrote some documentation

**Description**

### CODE REVIEW

File: db/r2yara [hunk 1]
```diff
@@ -1,6 +1,6 @@
 R2PM_BEGIN
 
-R2PM_GIT "https://github.com/radareorg/radare2-extras"
+R2PM_GIT "https://github.com/Yara-Rules/r2yara.git"
```
Comment (trufae) [inline db/r2yara:3]: Thats the wrong link

Comment (trufae) [inline db/r2yara:3]:
```suggestion
R2PM_GIT "https://github.com/radareorg/r2yara"
```

Comment (trufae) [reply db/r2yara:3]: That should work but i didnt tested it

Comment (sylvainpelissier) [reply db/r2yara:3]: Ah sorry I was also testing something else.

Comment (sylvainpelissier) [reply db/r2yara:3]: I tested, works fine.

**Comment by trufae** [comment]
I dont think ot will work because the install script is doing a cd r2yara that may fail

**Comment by sylvainpelissier** [comment]
It worked because of the fallback `cd`command but I removed it.

**Comment by trufae** [comment]
i merged it and it was failing. fixing now

**Pull request merged**